internal/services/player/delete.go

```diff
@@ -1,8 +1,42 @@
 package player
 
-import "github.com/jairogloz/go-l/internal/domain"
+import (
+	"log"
 
-func (s *Service) Delete(id domain.Player) (err error) {
-	//TODO implement me
-	panic("implement me")
+	"github.com/jairogloz/go-l/internal/domain"
+)
+
+// Delete player by id
+func (s *Service) Delete(id string) (err error) {
+	err = s.Repo.Delete(id)
+	if err != nil {
+		return manageError(err)
+	}
+	return nil
+}
+
+func manageError(err error) error {
```

Comment:
Buena propuesta! 👏 

Creo que podríamos hacer esta función más genérica y moverla a domain.app_error para que nos sirva no solo para delete palyer sino para todos los servicios.

Para esto también tenddríamos que hacer los mensajes más genéricos, por ejemplo, en lugar de "error deleting player: incorrect id" podría ser "incorrect id", y en lugar de "error deleting player: player not found" solo "not found". El punto es que los mensajes sean lo suficientemente genéricos para que nos sirvan para todos los servicios.

Luego podríamos parametrizar un poco y además del error recibir un `msg string` y si ese parámetro viene diferente de "" se puede concatenar con el mensaje genérico del error. 
```suggestion
func manageError(err error, msg string) error {
  var appErr domain.AppError
  switch err {
  case domain.ErrIncorrectID:
	log.Println("incorrect id error")
	appErr = domain.AppError{
		Code: domain.ErrCodeInvalidParams,
		Msg:  "error deleting player: incorrect id",
	}
	...
	}
	
	if msg != "" {
	  appErr.message := fmt.Sprintf("%s: appErr.message", msg)
	}
	return appErr
	
}
```